redash/query_runner/elasticsearch.py

```diff
@@ -35,7 +37,7 @@
 
 PYTHON_TYPES_MAPPING = {
     str: TYPE_STRING,
-    unicode: TYPE_STRING,
+    text_type: TYPE_STRING,
```

Comment:
BTW, we should leave a note to remove this duplicate-on-Python-3 thing when we switch to Python 3.

Reply:
This is super simple...  Just do a global find on __six__.  I have done this now on many projects and it is quite simple to do even across complex repos.  In fact, this is exactly why __six__ did not just reuse the names __unicode__ and __basestring__ on Python 3 but instead used different symbols that are easy to search on.  When __six__ is removed from the dependencies list these just pop right out on the automated testing.

Reply:
Makes sense, you're right. I've done it as well, and just tried to optimize for our future selves. Moving on.. :)